virtool/jobs/data.py

```diff
@@ -136,29 +175,43 @@ async def _find_beta(self, query: MultiDictProxy) -> Document:
                             "state": True,
                             "user": True,
                             "workflow": True,
-                        }
-                    },
-                ],
-            )
-        ]
+                        },
+                        "total_count": {
+                            "$arrayElemAt": ["$total_count.total_count", 0]
+                        },
+                        "found_count": {
+                            "$arrayElemAt": ["$found_count.found_count", 0]
+                        },
+                    }
+                },
+            ],
+        ):
+            data = paginate_dict["data"]
+            documents = [base_processor(document) for document in data]
+            try:
+                found_count = paginate_dict["found_count"]
+            except KeyError:
+                found_count = 0
```

Comment:
Is a `KeyError` possible here? I'd expect that the key is always defined.

Otherwise, cleaner to just use `paginate_dict.get("found_count", 0)`.

Same for `total_count`.

Reply:
The keys weren't defined whenever no data matched the query.